inst/tests/tests.Rraw

```diff
@@ -9122,6 +9122,10 @@ if (.Platform$OS.type=="unix") {
     unlink(f)
 }
 
+# Fix #1852 -- calling print on null data.table output NULL
+test(1704, dput(capture.output(print(data.table:::null.data.table()))),
```

Comment:
Better to avoid testing `:::` as this isn't user api, `data.table(NULL)` will works here. Also `dput` seems redundant.